resource_control: add note for the conflict between resource control … (#16563)
pingcap/docs: faq/sql-faq.md

- `DELAYED`: this statement has normal priority and is the same as the `NO_PRIORITY` setting for `tidb_force_priority`.

> > **NOTE:**

Reviewer: ```suggestion
> **Note:**
```

>

> > TiDB started supporting the feature [Resource Control](/tidb-resource-control.md) in v6.6.0. Users can utilize this feature to execute SQLs with different priorities under different resource groups. By configuring proper quotas and priorities for these resource groups, users can gain better scheduling control for SQLs with different priorities. When resource control is enabled, statement priority will no longer take effect. We recommend using [Resource Control](/tidb-resource-control.md) to manage the resource usage between different SQLs.

Reviewer: ```suggestion
> Starting from v6.6.0, TiDB supports [Resource Control](/tidb-resource-control.md). You can use this feature to execute SQL statements with different priorities in different resource groups. By configuring proper quotas and priorities for these resource groups, you can gain better scheduling control for SQL statements with different priorities. When resource control is enabled, statement priority will no longer take effect. It is recommended that you use [Resource Control](/tidb-resource-control.md) to manage resource usage for different SQL statements.
```
